docs/contribution.md

```diff
@@ -0,0 +1,80 @@
+
+Then the environment must be set up. This can be done by navigating to the openscope_databook directory and running `pip install -e . --user`.
+
+Finally, notebooks can be run with Jupyter notebook by running `jupyter notebook ./docs`
+
+
+### Make Contributions
+A contribution can take the form of changes to an existing notebook. More likely, it will be in adding a new analysis notebook to the databook. First, determine which *chapter* it will go into within the `./docs` directory, and create a new `.ipynb` file there. Then the notebook can be produced with the analysis and reproducible figures. For a contribution to be accepted it must adhere to the contribution **Styleguide** below. In addition to having clean code and clear figures, it is important that the notebook has thorough markdown explaining the figures and the analysis performed.
+
+**Important Note:**
+For a notebook to be compliant with the Databook, it must have an environment setup cell at the top. This is described in the **Styleguide** below.
+
+Any additional packages required for the notebook to run should be added to `requirements.txt`. When the notebook is complete it must be added to `docs/_toc.yml` to the appropriate chapter
+
+
+### Run and Test Locally
+Prior to pushing, the notebook should be tested locally. It should be ensured that it runs from beginning to end without interruption and produces clean-looking figures. To confirm everything is configured properly, the Jupyter book should be built and examined with the command `jupyter-book build ./docs`.
+
+
+### Commit, Push and Make a Pull Request
+Local progress can be saved with `git commit` and then pushed to the remote fork with `git push`. See this [Git tutorial](https://www.atlassian.com/git/tutorials/saving-changes/git-commit) for more information on how to do this. 
+
+A PR can then be made to the original OpenScope Databook repo. On the forked repo front page, first make sure to press the `sync fork` button, and then `contribute` can be pressed to make a PR. It is advised to make the PR into the Databook's dev branch rather than the main branch.
+
+
+### Interact on ReviewNB
+Once the pull request is made, it will be reviewed for merging into the Databook. This process can be monitored on the PR page, where there will be a purple button linking to ReviewNB. ReviewNB allows for the additions and changes made to notebooks to be easily viewed and commented on. Please check this for any feedback provided by the Databook team and make revisions accordingly. Once the PR is merged, you will be added to the authors list of the Databook.
+
```

Comment:
ADD: 

### Authorships
Authorship will be given upon merging of any PR to the main branch. A script running on building of the repository extract the list of GIT contributors and automatically add a git contributor name. The following file (ADD link here) allows to modify your GIT username to reflect your real name. Please feel free to make a commit to correct your name in this file  if you desire to be listed differently than your GIT username.